hugo/layouts/community/baseof.html

```diff
@@ -16,6 +16,7 @@
           <div class="resize-handle--x" data-target="aside"></div>
           <main class="col-12 col-md-9 col-xl-8 pl-md-5" role="main">
             {{ partial "version-banner.html" . }}
+            {{ partial "community-bio.html" . }}
```

Comment:
Can you make it as a shortcode part of the md page instead of a partial? This as being a partial we will need to maintain the `baseof.html` layout